Userland/Libraries/LibWeb/DOM/StaticRange.idl

```diff
@@ -2,14 +2,14 @@
 #import <DOM/AbstractRange.idl>
 
 // https://dom.spec.whatwg.org/#staticrange
```

Comment:
The spec links are a bit off here now - this one belongs over `StaticRange`, and and `https://dom.spec.whatwg.org/#dictdef-staticrangeinit` goes over `StaticRangeInit` (although we didn't have the latter at all before)

Reply:
Good catch - did not know that individual definitions had their own permalinks, which makes sense. We're probably missing those in a lot of places.